experimental/packages/opentelemetry-sdk-node/src/sdkNode.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  ConfigFactory,
+  createConfigFactory,
+} from '@opentelemetry/configuration';
+import { SDKConfiguration } from './types';
+import { diag, DiagConsoleLogger } from '@opentelemetry/api';
+import { setupContextManager } from './utils';
+
+/**
+ * Experimental function to start the OpenTelemetry Node SDK
+ * @param configurationSDK
+ */
+export const startNodeSDK = (
+  configurationSDK: Partial<SDKConfiguration> = {}
```

Comment:
naming-nit (i.e. big alarm bells that I'm choose bikeshed colours here): This is a pitch for using a name other than "configuration" for the params to `startNodeSDK` to try to somewhat disambiguate from the "Configuration"-things being specified by the OTel Configuration spec and the configuration package.

- call the type `SDKOptions`
- call the parameter `opts` or `sdkOptions` or `sdkOpts`